firefox-ios/firefox-ios-tests/Tests/ClientTests/Telemetry/HomepageTelemetryTests.swift

```diff
@@ -11,30 +10,54 @@ final class HomepageTelemetryTests: XCTestCase {
     override func setUp() {
         super.setUp()// Due to changes allow certain custom pings to implement their own opt-out
         // independent of Glean, custom pings may need to be registered manually in
```

Comment:
We should remove the comment in this section since its not relevant anymore

Reply:
I've removed an unnecessary overridden function in HomepageTelemetryTests.swift to resolve a warning about unneeded overrides

Reply:
is it good to merge now?